pkg/setting/setting_unified_alerting.go

```diff
@@ -89,6 +90,41 @@ type UnifiedAlertingScreenshotSettings struct {
 	UploadExternalImageStorage bool
 }
 
+type UnifiedAlertingHistoryStorage string
```

Comment:
Remove these - we're only using a "DemoEnabled" setting from now. These are left over from a previous prototyping pass.